source/about/on-prem-skype-for-business-replacement.rst

```diff
@@ -0,0 +1,51 @@
+
+The following mission-ready collaboration capabilities are available: 
+
+Air-Gapped and Classified Operations
+------------------------------------
+
+Organizations operating in fully disconnected or classified environments require secure communication platforms that function entirely within their own infrastructure.
+
+**Benefits**
+
+- **Ensure secure communication in fully disconnected networks** using Mattermost’s support for private on-premise deployments, including FIPS 140-2 validated and DISA STIG-hardened container images.
+- **Maintain operational continuity with enterprise-grade collaboration tools**—including :doc:`1:1 audio calls </collaborate/make-calls>`, :ref:`screen sharing <collaborate/make-calls:share your screen>`, :doc:`threaded messaging </collaborate/organize-conversations>`, and :doc:`file sharing </collaborate/share-files-in-messages>`—entirely within air-gapped systems.
+- **Scale to mission requirements** with a :doc:`high-availability, horizontally scalable architecture </scale/scaling-for-enterprise>` that supports tens of thousands of users in secure on-prem environments.
+- **Preserve data sovereignty and eliminate external dependencies** with a self-hosted :doc:`Kubernetes deployment model </deploy/server/deploy-kubernetes>` that integrates into classified networks or sovereign data centers.
+
+Modernize Secure Collaboration Workflows
+------------------------------------------
+
+Legacy communication tools lack the flexibility, automation, and usability demanded by modern operational teams. Mattermost introduces modern collaboration workflows without compromising compliance or deployment control.
+
+**Benefits**
+
+- **Enable dynamic, cross-platform messaging and coordination** with a unified interface across web, desktop, and mobile—featuring :doc:`threaded discussions </collaborate/organize-conversations>`, :ref:`file previews <collaborate/share-files-in-messages:preview file attachments>`, and :ref:`screen sharing <collaborate/make-calls:share your screen>`.
+- **Streamline mission-critical processes** with :doc:`Collaborative Playbooks </guides/repeatable-processes>` that automate and track workflows like incident response, shift turnover, and logistics planning.
+- **Embed secure video conferencing into daily operations** using Pexip integration, allowing real-time video engagement from within your air-gapped or secure infrastructure.
```

Comment:
We should add docs; can be task for a later date, and add this link for now: https://mattermost.com/marketplace/pexip-video-connect/

Reply:
Also logged https://mattermost.atlassian.net/browse/CX-244 to capture the existing docs gap.